draft-ietf-lamps-csr-attestation.md

```diff
@@ -499,24 +500,28 @@ that multiple Verifiers are registered against the same type OID in which case t
 Relying Party will either require additional parsing of the evidence statement, or
 the Attester will be required to provide additional information.
 
-To simplify the task for the Relying Party an optional field, the hint, is available
-in the EvidenceStatement structure, as shown in {{code-EvidenceStatement}}. An
-Attester MAY include the hint to the EvidenceStatement and it MAY be processed
-by the Relying Party. The Relying Party MAY decide not to trust the information
-embedded in the hint or policy MAY override any information provided by the Attester
-via this hint.
-
-When the Attester populates the hint, it MUST contain a fully qualified domain
-name (FQDN) which uniquely identifies a Verifier.
-The problem of mapping hint FQDNs to Verifiers, and the problem of FQDN collision
-is out of scope for this specification; it is assumed that Attester and Verifier
-makers can manage this appropriately on their own FQDN trees, however if this
-becomes problematic then a public registry may be needed.
+To simplify the task for the Relying Party to select an appropriate Verifier
+an optional field, the hint, is available in the EvidenceStatement structure,
+as shown in {{code-EvidenceStatement}}. An Attester MAY include the hint to
+the EvidenceStatement and it MAY be processed by the Relying Party. The
+Relying Party MAY decide not to trust the information embedded in the hint
+or policy MAY override any information provided by the Attester via this hint.
+
+When the Attester populates the hint, it MUST contain a server name which
+uniquely identifies a Verifier. Server names are ASCII strings that
+contain a hostname and optional port, where the port is implied to be
+"443" if missing.  The names use the format of the authority portion
+of a URI as defined in Section 3.2 of {{RFC3986}}. The names MUST NOT
+include a "userinfo" portion of an authority.  For example, a valid
+server name might be "verifier.example.com" or
+"verifier.example.com:8443", but not "verifier@example.com".
```

Comment:
I would add a sentence:

"Relying Parties SHOULD NOT connect to a host name provided in the hint, especially if the verifier has no previous trust relationship with that host name, instead this SHOULD be used only as a lookup string for determining between a list of Verifiers that the Relying Party is pre-configured to use."